tests/perf/test_init_time.py

```
@@ -1,14 +1,62 @@
+            check=True,
+        )
+        samples.append(float(result.stdout))
+    return min(samples)
+
+
+def test_cli_time() -> None:
     if shutil.which("celesto") is None:
         pytest.skip("celesto CLI is provided by the separate celesto package.")
     t0 = time.perf_counter()
-    assert subprocess.check_call(["celesto", "--help"]) == 0
+    subprocess.check_call(["celesto", "--help"])  # raises on failure; see above
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)